operator/images/cluster-setup/content/bin/install.sh

```diff
@@ -150,6 +150,13 @@ install_shared_manifests() {
 install_applications() {
   CONFIG_DIR=$(find "${WORKSPACE_DIR}/environment/compute" -type d -name "${clusters[$i]}")
   kubectl apply -k "$CONFIG_DIR"
+  if [ -n "${GIT_URL:-}" ] || [ -n "${GIT_REF:-}" ];then
```

Comment:
i have a feeling this could break things, but since it's not used to deploy user facing environments anymore, I'm fine with the change.

Reply:
Might this also cause issues with the `--use-current-branch` flag? It might also conflict a bit with https://github.com/openshift-pipelines/pipeline-service/pull/582.

I suggest that we consider changes related to the `GIT_URL` and `GIT_REF` variables in a separate pull request.

Reply:
> I suggest that we consider changes related to the GIT_URL and GIT_REF variables in a separate pull request.

Ok I will work on that and see if hopefully this PR won't be dependent on that

Reply:
Done in https://github.com/openshift-pipelines/pipeline-service/pull/588